include/libswiftnav/track.h

```diff
@@ -121,11 +121,14 @@ typedef struct {
 typedef struct {
   float log_bw;     /**< Noise bandwidth in dBHz. */
   float b;          /**< IIR filter coeff. */
-  float a;          /**< IIR filter coeff. */
+  float a2;         /**< IIR filter coeff. */
```

Comment:
ctypedef struct cn0_est_state_t in libswiftnav/python/swiftnav/track.pxd needs to be updated to contain these new fields. It is needed for Peregrine.